.cloudbuild/README.md

```diff
@@ -14,3 +14,28 @@ new image to Google Kubernetes Engine cluster, then run all tests in
   unit tests in `keras_nlp/`.
 
 This test is powered by [ml-testing-accelerators](https://github.com/GoogleCloudPlatform/ml-testing-accelerators).
+
+### Adding Test Dependencies
+You must be authorized to run builds in the `keras-team-test` GCP project.
+If you are not, please open a GitHub issue and ping a team member.
+To authorize yourself with `keras-team-test`, run:
+
+```bash
+gcloud config set project keras-team-test
+```
+
+To add/update dependency for GPU tests:
+- Add/update dependency to `requirements.txt`
+- Create a `Dockerfile` with the following contents:
+```
```

Comment:
newline before and after codeblocks, I think adding dockerfile after the backticks will add syntax highlighting.

And if you indent the code block to line up with "Create" if will actually indent to the code block to be part of the list.